core/consensus/grandpa/impl/voting_round_impl.cpp

```diff
@@ -225,7 +225,7 @@ namespace kagome::consensus::grandpa {
     pending_timer_handle_ = scheduler_->scheduleWithHandle(
         [&] { pending(); },
         toMilliseconds(std::max<Clock::Duration>(duration_ * 10,
-                                                 std::chrono::seconds(30))));
+                                                 std::chrono::seconds(12))));
```

Comment:
What about taking this to a named constant? (looking at the number of CLI parameters in substrate, maybe even a CLI parameter)